rbac/management/role/definer.py

```diff
@@ -23,6 +23,7 @@
 from django.conf import settings
 from django.db import transaction
 from django.utils import timezone
+from internal.views import destructive_ok
```

Comment:
Small suggestion would be we might want to keep the concerns separated here so that the role definer doesn't need to know about internal views? It would probably mean we pull it out into a common/shared `utils` module, which we don't currently have. Curious what your thoughts are? There are probably other candidates for things that we're currently duplicating across some of the app-specific `utils` modules. Maybe it's outside the scope of this?

Reply:
I added #747 which would address this.

Reply:
I think it's a good idea, though likely out of the scope of this particular change.